Cargo.toml

```diff
@@ -52,13 +52,21 @@ path = "./postcard-derive"
 version = "0.1.1"
 optional = true
 
+[dependencies.crc]
+version = "3.0.1"
+optional = true
+
+[dependencies.paste]
+version = "1.0.12"
```

Comment:
For some help with our declarative macros.

Reply:
Would it make sense to make this optional and gate it on the crc feature?

Reply:
We could, although perhaps that adds a little complexity in case we need it elsewhere?

Alternatively, let's get rid of the macros and the need for this. :-)

Reply:
Commit 2c9ed556ff3ce385119932e0b5c61b83519c9c62 makes the `paste` crate optional and includes it when `use-crc` is enabled.